### PR TITLE
feat(apps/gql): add post upvote mutations

### LINKED ISSUE #554: feat(gql): Create `createPostUpvote` mutation
Use https://github.com/kamp-us/monorepo/pull/572/commits/214d224657a6012770b9e2e65619953c088ccf48 as an example

### LINKED ISSUE #556: feat(gql): Create `removePostUpvote` mutation
Use https://github.com/kamp-us/monorepo/pull/572/commits/214d224657a6012770b9e2e65619953c088ccf48 as an example

### PR DESCRIPTION
**Pull request by rasitds**
# Description

Created create and remove mutations for post upvotes.
Also ensured that postID and userID are unique with one block attribute (@@unique) in the psima schema.

Closes #554 
Closes #556

### CODE REVIEW

**Review by FurkanGM — APPROVED** [review]
LGTM 💯

File: apps/gql/schema/resolvers/index.ts [hunk 1]
```diff
@@ -347,5 +351,37 @@ export const resolvers = {
...
       return transformPanoComment(await actions.pano.comment.remove(id.value));
     },
+    createPostUpvote: async (_, { input }, { actions, pasaport: { session } }) => {
+      if (!session?.user?.id) {
+        return NotAuthorized();
+      }
+
+      if (!input.postID) {
+        return InvalidInput("post is required");
+      }
```
Comment (Ketcap) [inline apps/gql/schema/resolvers/index.ts]: should this be already required on graphql ?

File: apps/gql/schema/resolvers/index.ts [hunk 2]
```diff
@@ -347,5 +351,37 @@ export const resolvers = {
...
+      );
+    },
+    removePostUpvote: async (_, { input }, { actions, pasaport: { session } }) => {
+      if (!session?.user?.id) {
+        return NotAuthorized();
+      }
+
+      if (!input.postID) {
+        return InvalidInput("post is required");
+      }
```
Comment (Ketcap) [inline apps/gql/schema/resolvers/index.ts]: same required like above

**Review by Ketcap — APPROVED** [review]

File: apps/gql/schema/resolvers/index.ts (hunk 1 above)
Comment (usirin) [reply apps/gql/schema/resolvers/index.ts]: yep, this is defined as required in the schema, so we can remove this.

File: apps/gql/schema/resolvers/index.ts (hunk 2 above)
Comment (usirin) [reply apps/gql/schema/resolvers/index.ts]: same, let's delete this.

**Review by usirin — APPROVED** [review]
💯 <3

**Pull request merged**